src/components/OrgFile/components/Header/index.js

```diff
@@ -361,6 +361,21 @@ ${header.get('rawDescription')}`;
           headerDeadlineMap.get('year')
         : '';
 
+    const today = new Date();
+    const headerDate =
+      headerDeadlineMap !== undefined
+        ? new Date(
+            headerDeadlineMap.get('month') +
+              '-' +
+              headerDeadlineMap.get('day') +
+              '-' +
+              headerDeadlineMap.get('year')
+          )
+        : today;
```

Comment:
I don't understand how you construct this `new Date()`.
When I enter this weird order of arguments in node:
```js
> Date('01-05-2022')
'Wed Nov 29 2023 21:54:27 GMT+0100 (Central European Standard Time)'
```
I'd suggest to pass a date in ISO format.